packages/react/ds-core-form/package.json

```diff
@@ -33,7 +33,8 @@
     "test": "bun run test:vitest",
     "test:watch": "bun run test:vitest:watch",
     "test:vitest": "vitest run",
-    "test:vitest:watch": "vitest"
+    "test:vitest:watch": "vitest",
+    "init:msw": "npx msw init public/"
```

Comment:
I don't see this command executed anywhere, is it "magic" invoked by the worker script?

Reply:
It would be nice to include a note in the README about this command and what scenarios we might need to re-run it in. For instance, if upgrading versions of the MSW script. That way, the relationship between this command, the worker file, and the package are a bit more clear to somebody with less exposure to MSW.